src/types.js

```diff
@@ -2,6 +2,7 @@
 import type { IntlShape } from 'react-intl';
 import type { DangerouslyImpreciseStyleProp } from 'react-native/Libraries/StyleSheet/StyleSheet';
 
+import type { SubsetProperties } from './generics';
 import type { Auth, Topic, Message, PmRecipientUser, Reaction, ReactionType } from './api/apiTypes';
```

Comment:
d220cc82f types: Use SubsetProperties to base Outbox more directly on Message.

`tools/test lint` tells me `Reaction` and `PmRecipientUser` are defined but never used, in this commit.

Reply:
> `tools/test lint` tells me `Reaction` and `PmRecipientUser` are defined but never used, in this commit.

Ah, thanks for the catch! Fixed.